README.md

```diff
@@ -80,11 +80,15 @@ OpenCL is used to improve performance in different platforms.
 
         export LD_LIBRARY_PATH=/usr/local/lib/:$LD_LIBRARY_PATH
         export GST_PLUGIN_PATH=/usr/local/lib/gstreamer-1.0:$GST_PLUGIN_PATH
+        export OPENVINO_EXTRAS_PATH=WORKDIR/openvino/
+        source /usr/local/setupvars.sh
 
     --prefix=/usr:
 
         export LD_LIBRARY_PATH=/usr/lib/:$LD_LIBRARY_PATH
         export GST_PLUGIN_PATH=/usr/lib/gstreamer-1.0:$GST_PLUGIN_PATH
+        export OPENVINO_EXTRAS_PATH=WORKDIR/openvino/
+        source /usr/setupvars.sh
```

Comment:
shell "setupvars.sh" are commonly used for many 3rd party libraries. put this openvino speicified shell under folder /usr might be not very friendly.

Reply:
same as line 91, for example:
/opt/intel/openvino_2023/setupvars.sh
/opt/intel/oneapi/setupvars.sh
... ...